tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AWS/DataStreamsMonitoringAwsSqsTests.cs

```diff
@@ -0,0 +1,97 @@
+// <copyright file="DataStreamsMonitoringAwsSqsTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading.Tasks;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.TestHelpers;
+using Datadog.Trace.TestHelpers.DataStreamsMonitoring;
+using FluentAssertions;
+using VerifyXunit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.AWS;
+
+[Collection(nameof(AwsSqsTests.AwsSqsTestsCollection))]
+[Trait("RequiresDockerDependency", "true")]
+[UsesVerify]
+public class DataStreamsMonitoringAwsSqsTests : TestHelper
+{
+    public DataStreamsMonitoringAwsSqsTests(ITestOutputHelper output)
+        : base("AWS.SQS", output)
+    {
+    }
+
+    public static IEnumerable<object[]> GetEnabledConfig()
+    {
+        return from packageVersionArray in PackageVersions.AwsSqs
+               select new[] { packageVersionArray[0] };
+    }
+
+    [SkippableTheory]
+    [MemberData(nameof(GetEnabledConfig))]
+    [Trait("Category", "EndToEnd")]
+    public async Task SubmitsDsmMetrics(string packageVersion)
```

Comment:
The `DataStreamsMonitoringTests` tests include a comment that describes what we _expect_ to see - would it be possible to add that, so that it gives something to compare the snapshots against 🙂

Reply:
I wrote a little something to explain what happens in the sample app, but I'm not going to write anything as detailed as in the kafka test.
I derived my test from the AwsSqsTest, which has zero explanation, so I can write what I understood from reading the sample program, but writing a full explanation is beyond the scope of that PR I think.